packages/loader/container-loader/src/loader.ts

```diff
@@ -357,8 +357,19 @@ export class Loader implements IHostLoader {
 		return this;
 	}
 
-	public async createDetachedContainer(codeDetails: IFluidCodeDetails): Promise<IContainer> {
-		const container = await Container.createDetached(this.services, codeDetails);
+	public async createDetachedContainer(
+		codeDetails: IFluidCodeDetails,
+		containerCreateHeaders?: IRequestHeader,
```

Comment:
lets avoid using header here, as we are trying to move away from request like patterns, i would just take an object, something like:
```suggestion
		props?: {
		canReconnect?: boolean,
		clientDetailsOverride?: IClientDetails
		}
```